src/client/components/map-box/map-box.vue

```diff
@@ -426,6 +426,7 @@ export default {
           const layers = this.map.getStyle().layers
           const lastWmsLayerIndex = layers
             .filter(layer => /wms-layer-/.test(layer.id))
+            .reverse()
```

Comment:
I thought the initial order (without reversing it) was good - the bottom one in the layer list should be the _first_ drawn so it appears at the bottom of the mapbox layer stack as well

Reply:
ok it appears that after a rebuild it works fine